node/src/blockchain/blockchain_interface.rs

```diff
@@ -753,7 +755,13 @@ mod tests {
             &Wallet::from_str("0x3f69f9efd4f2592fd70be8c32ecd9dce71c472fc").unwrap(),
         );
 
-        assert_eq!(Err(BlockchainError::QueryFailed), result);
+        assert_eq!(
```

Comment:
If you don't like being this specific about the error message, you could try something like

match result {
    Err(BlockchainError::QueryFailed(msg)) if msg.contains ("Invalid hex character: Q") => (),
    x => panic! ("Expected complaint about hex character, but got {:?}", x)
}